libr/bin/bfile.c

```diff
@@ -953,67 +867,12 @@ R_IPI RList *r_bin_file_get_strings(RBinFile *a, int min, int dump, int raw) {
 		}
 	} else {
 		if (a) {
-			r_bin_file_get_strings_range (a, ret, min, raw, 0, a->size);
```

Comment:
Ideally this functionality should be implemented in rsearch. But this change doesnt fits in this pr. Just keep it in mind for future changes

Reply:
ok good to know